sql/dql/database-clients/node-database-packages.md

```diff
@@ -36,34 +36,34 @@ aspects:
 ---
 ## Content
 
-Node is another popular platform (available as a course for premium users) which uses Java Script. As there are lots of packages available for it we will take a look at some of the database packages:
+Node is another popular platform (available as a course for premium users) which uses JavaScript. As there are lots of packages available for it we will take a look at some of the database packages:
```

Comment:
The premium-ness should be dynamically explained and we can remove it from static content:

```suggestion
Node is another popular platform and it uses JavaScript. As there are lots of packages available for it we will take a look at some of the database packages:
```